roles/gitea/defaults/main.yml

```diff
@@ -0,0 +1,79 @@
+---
+gitea_user: gitea
+gitea_group: gitea
+gitea_home: "/var/lib/gitea"
+gitea_shell: "/bin/false"
+gitea_version: "1.20.3"
+gitea_version_check: true
+gitea_config_dir: "/etc/gitea"
+
+gitea_dl_url: "https://github.com/go-gitea/gitea/releases/download/v{{ gitea_version }}/gitea-{{ gitea_version }}-linux-{{ gitea_arch }}"
+gitea_repository_root: "{{ gitea_home }}"
+gitea_user_repo_limit: -1
+
+gitea_app_name: "Metal Gitea"
+gitea_http_domain: localhost
+gitea_root_url: http://localhost:3000
+gitea_protocol: http
+gitea_http_listen: 0.0.0.0
+gitea_http_port: 3000
+gitea_disable_http_git: false
+gitea_offline_mode: true
+
+gitea_db_type: sqlite3
+gitea_db_path: "{{ gitea_home }}/data/gitea.db"
+
+gitea_ssh_listen: 0.0.0.0
+gitea_ssh_domain: gitea
+gitea_start_ssh: true
+gitea_ssh_port: 2222
+
+gitea_secret_key: T0pS3cr31
+gitea_internal_token: SomethingVeryLong
+gitea_disable_git_hooks: true
+
+gitea_show_user_email: false
+gitea_disable_gravatar: true
+gitea_disable_registration: true
+gitea_show_registration_button: false
+gitea_require_signin: false
+gitea_enable_captcha: false
+gitea_only_allow_external_registration: false
+gitea_enable_notify_mail: false
+gitea_mail_default: onmention
+gitea_autowatch_new_repo: false
+gitea_autowatch_on_change: true
+gitea_show_mailstones_dashboard: true
+
+gitea_force_private: false
+
+gitea_mailer_enabled: false
+gitea_mailer_skip_verify: false
+gitea_mailer_tls_enabled: true
+gitea_mailer_host: localhost:25
+gitea_mailer_from: noreply@your.domain
+gitea_mailer_user: ""
+gitea_mailer_password: ""
+gitea_mailer_type: smtp
+
+gitea_oauth2_enabled: false
+gitea_oauth2_jwt_secret: ''
+
+gitea_metrics_enabled: false
+gitea_metrics_token: ~
+
+gitea_themes: gitea,arc-green
+gitea_theme_default: gitea
+
+gitea_repo_indexer_enabled: false
+gitea_repo_indexer_include: ""
+gitea_repo_indexer_exclude: ""
+gitea_repo_exclude_vendored: true
+gitea_repo_indexer_max_file_size: 1048576
+
+gitea_log_level: Info
+
+gitea_extra_config: ""
+
+gitea_backup_on_upgrade: false
+gitea_backup_location: "{{ gitea_home }}/backups/"
```

Comment:
We should make a JIRA for mounting a backup volume or something here, so we can more easily reattach the gitea stuff to a new management-vm.

Reply:
https://jira-pro.it.hpe.com:8443/browse/MTL-2263